vllm/model_executor/models/minicpmv.py

```diff
@@ -400,13 +405,11 @@ def get_embedding(
         return vlm_embedding, vision_hidden_states
 
     def _get_image_bounds(self, input_ids: torch.Tensor) -> torch.Tensor:
-        tokenizer = cached_get_tokenizer(self.config._name_or_path,
-                                         trust_remote_code=True)
-        start_cond = input_ids == tokenizer.im_start_id
-        end_cond = input_ids == tokenizer.im_end_id
-        if hasattr(tokenizer, "slice_start_id"):
-            start_cond |= (input_ids == tokenizer.slice_start_id)
-            end_cond |= (input_ids == tokenizer.slice_end_id)
```

Comment:
Can we pass this information via the multi-modal kwargs?

Reply:
You mean MultiModalConfig?

Reply:
I mean that apart from the image input data (e.g. `pixel_values`), we may be able to return these IDs via the input processor which does have access to the tokenizer.

Reply:
Nice work, this is much cleaner now!